testing/env_yamls/has_conda.yml

```diff
@@ -3,4 +3,3 @@ name: has_conda
 dependencies:
     - conda
     - toolz
-    - python=3.7
```

Comment:
This allows the current `python 3.8.5` package to get pulled in, which is one of those packages affected. Eventually a fixed version of the python package will replace it in the testbed and we won't be exercising this fix anymore.